modules/mono/editor/GodotTools/GodotTools.ProjectEditor/ProjectGenerator.cs

```diff
@@ -8,7 +8,7 @@ namespace GodotTools.ProjectEditor
 {
     public static class ProjectGenerator
     {
-        public const string GodotSdkVersionToUse = "3.2.4";
+        public const string GodotSdkVersionToUse = "3.3";
```

Comment:
The Mono changes are not really needed. We don't have to bump the packages with each release. That said, if you still want, I can publish the new version of the packages to NuGet; the changes are right.

Reply:
Yeah I think it would be clearer to users if we do the update to NuGet too. There's going to be a fair amount of confusion with t he rename from 3.2.4 to 3.3 already :P

Reply:
I don't see why we wouldn't bump the version, it would be confusing if 3.3 was using "3.2.4" for something.

Reply:
If we want to keep them decoupled the GodotSdk version should likely use a different versioning scheme. It started in sync with Godot but since version bumps are not needed the version number will become confusing over time.

Could be kept in sync with Godot `major.minor` and then with an additional GodotSdk specific version number? E.g. `3.3.a` if NuGet supports letters.

Reply:
Let's update the version, but only for `3.3` and not minor versions (unless there are actual changes, which is unlikely for the 3.x branch). I don't think that would cause confusion.

For 4.0 or 4.1 the idea is to start shipping the C# API as a NuGet package and also get rid of the custom Sdk by moving all its build system files to the C# API package itself. The only reason we use a custom Sdk right now is because it allows us to tell MSBuild to use the `./.godot/mono/temp/bin` and `./.godot/mono/temp/obj` directories instead of `./bin` and `./obj`. Everything else can be done with a normal package.
At that point we will be syncing versions for that package and the publishing to NuGet should be automatic in the official releases build scripts.

Reply:
Apparently `<Project Sdk="Godot.NET.Sdk/3.3">` results in a build warning even though it still resolves to `3.3.0` correctly... Not critical but we need to change `GodotSdkVersionToUse` to `3.3.0` to get rid of the warning. The package is published now: https://www.nuget.org/packages/Godot.NET.Sdk/3.3.0

Reply:
Fixed with e6b291b152.